src/Pelagos/Bundle/AppBundle/Controller/UI/AccountController.php

```diff
@@ -155,7 +155,7 @@ public function sendVerificationEmailAction(Request $request)
      * This verifies that the token has authenticated the user and that the user does not already have an account.
      * It then provides the user with a screen to establish a password.
      *
-     * @Route("/verify-email")
+     * @Route("/account/verify-email")
```

Comment:
For some reason this is causing the token to not be recognized on a password reset action, resulting in the "The link you have selected is invalid. The link may have already been used to create an account. If you have forgotten your password please visit password reset. For any additional help please contact griidc@gomri.org. " error.

Reply:
Fixed by removing old %pelagos-prefix% in firewall.